src/devices/cpu/mc68hc11/mc68hc11.cpp

```diff
@@ -384,6 +397,71 @@ void mc68hc11_cpu_device::tmsk2_w(uint8_t data)
 	m_tmsk2 = data;
 }
 
+void mc68hc11_cpu_device::serial_w(u8 val)
+{
+	m_scdr = val;
+	m_scsr |= 0x20;
+	if ((m_sccr2 & 0x20) != 0)
+		set_irq_state(0x14, true);
+}
+
+// unimplemented
+uint8_t mc68hc11_cpu_device::baud_r()
+{
+	return m_baud;
+}
+void mc68hc11_cpu_device::baud_w(uint8_t data)
+{
+	m_baud = data;
+}
+
+// unimplemented
```

Comment:
If it's "unimplemented" then it needs logging in the handler, not being concealed like this (nobody will see it essentially).